src/index.ts

```diff
@@ -97,18 +95,18 @@ interface JosaCompleter {
  * Creates a {@link JosaCompleter} instance
  * @param whenTrue string to append when predicate returned true
  * @param whenFalse string to append when predicate returned false
- * @param customBranch takes the last letter of a word you're appending to
+ * @param customBranching takes the last letter of a word you're appending to
```

Comment:
not sure which is the best
- branch[ing]
- test[er]
- policy

Reply:
`customBranching` can be the proper one. I like it 😍